types/V1.mli

```diff
@@ -225,32 +252,32 @@ module type CONSOLE = sig
   *)
 end
 
+(** {1 Sector-addressible block devices} *)
+
+(** Block device errors. *)
 module Block : sig
-  type error = [
-    | `Msg of string     (** an unspecified error *)
-    | `Unimplemented     (** operation not yet implemented in the code *)
-    | `Disconnected      (** the device has been previously disconnected *)
-  ]
+
+  type error = Device.error
```

Comment:
wouldn't we want to be able to extend for a specific `Block` to include a `` `Msg of string ``!?